### DISCUSSION TITLE
MAS A exception has occurred

### DISCUSSION THREAD

**Discussion by eggywiggy** [Monika-After-Story/MonikaModDev#8901] [QA - tech support + general] [answered]
[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/sprite-decoder.rpy", line 2, in script
    init python in mas_sprite_decoder:
  File "game/sprite-decoder.rpy", line 321, in <module>
    _m1_sprite0x2ddecoder__loadSpriteMapData()
  File "game/sprite-decoder.rpy", line 71, in _m1_sprite0x2ddecoder__loadSpriteMapData
    raise MASSpriteException(repr(e) + "\n\nPLEASE REINSTALL MAS TO CORRECT THIS ERROR")
MASSpriteException: IOError(2, 'No such file or directory')

PLEASE REINSTALL MAS TO CORRECT THIS ERROR

I have reinstalled many times but still have the same error what should I do

**Comment by multimokia** [comment, marked as answer]
When reinstalling be sure to copy over the items from mod_assets as well. The json containing spritecode mappings is missing from your install
